boot/bootchain.go

```diff
@@ -37,6 +40,9 @@ type bootChain struct {
 	// kernel is unasserted, in which case always reseal.
 	KernelRevision string `json:"kernel-revision"`
 	KernelCmdline  string `json:"kernel-cmdline"`
+	model          *asserts.Model
+	kernelBootFile bootloader.BootFile
+	blName         string
```

Comment:
I don't think we want blName here,   we'll have to give the places that care some kind of map:

role -> blName